hue/README.md

```diff
@@ -0,0 +1,20 @@
+# HUE - Hadoop User Experience 
+
+This [initialization action] (https://cloud.google.com/dataproc/init-actions) installs the latest version of [HUE](http://gethue.com/) 
+on a master node within a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+
+## Using this initialization action
```

Comment:
I'm not sure what to add here.
I don't think I want to mention Livy because we couldn't make it work, and I checked to oozie script - it seems to be broken (regardless of the hue script).